### ISSUE TITLE
Defect: Major - Tabbed Module - Trucks: Volvo VNR Electric - Names of the Tabs on the Navigation bar are different; additional differences between text and title font size and underline margin

### ISSUE THREAD

**Issue by Andrei-Boiangiu** [hlxsites/vg-volvotrucks-us#190]
The names of Tabs on the Tabbed Module are different from the volvotrucks website ( ex: Configurations tab appears twice on hlx.page), additionally Title and Text font size on Tabs container is different as well as the margin of the underline on the selected Tab

**Steps to reproduce:**
01. Go to https://main--vg-volvotrucks-us--hlxsites.hlx.page/trucks/vnr-electric/ and scroll down to the Tabbed Module (Configurations, Charging etc tabs)
02. Go to https://www.volvotrucks.us/trucks/vnr-electric/ and scroll down to the Tabbed Module (Configurations, Charging etc tabs)
03. Notice the difference between the name of the tabs
04. Notice the font size on the tab's content for Title and Text
05. Notice the difference in the margin of the underline marking the selected tab

**Expected result**
Tabs should be **Configurations, Drivetrain** and **Charging**
Font size for Title and Text in the tab content should be smaller as well as the margin for the underline under the text of the selected tab
see right half of screenshot

![image (214)](https://user-images.githubusercontent.com/127409602/229530292-cfc1356e-fe02-4f0c-b138-bb2feec3d2af.png)

**Actual result**
Tabs are named Configurations, Charging and Configurations (Configurations is twice also)
Font size for Title and Text in the tab content is too large as well as the margin for the underline under the text of the selected tab
see left half of screenshot

 
![image (214)](https://user-images.githubusercontent.com/127409602/229530671-00dddcdc-83b7-4948-80d5-6733086f5b9e.png)

**Comment by andreibalanescu** [comment]
This looks already fixed. Please check and let me know.

**Comment by Andrei-Boiangiu** [comment]
this is still present as described 
but maybe can be tackled within the Font Size difference ( #209 ) and Font Styling for the underline margin ( #216 ) tickets?

<img width="1903" alt="image" src="https://user-images.githubusercontent.com/127409602/231216534-55d52fa1-6c81-4540-9939-514c95e80687.png">

**Comment by andreibalanescu** [comment]
Yes, indeed. The naming is correct now. Regarding other issues, let's see how it looks like when #209 and #216 will be fixed.
